ledger_device_sdk/src/ecc.rs

```diff
@@ -377,7 +377,12 @@ impl<const P: usize> ECPublicKey<P, 'E'> {
 ///
 /// Checks consistency of curve choice and key length
 /// in order to prevent the underlying syscall from throwing
-pub fn bip32_derive(curve: CurvesId, path: &[u32], key: &mut [u8]) -> Result<(), CxError> {
+pub fn bip32_derive(
+    curve: CurvesId,
+    path: &[u32],
+    key: &mut [u8],
+    cc: &mut [u8],
+) -> Result<(), CxError> {
```

Comment:
I don't get why for ed25519 the requested length is 96 few lines below?

Reply:
Agreed, looks to be ok with 64 🤔. Maybe a typo @yhql ?

Reply:
Fix in [this commit](https://github.com/LedgerHQ/ledger-device-rust-sdk/pull/132/commits/5b63e3e971e242e5a43304f4459c467374d23c67), waiting feedback from @yhql if revert is needed